server/api/cart.js

```diff
@@ -140,56 +125,89 @@ router.put('/:fruitId', async (req, res, next) => {
   }
 })
 
-// A PUT request???
+// A PUT route to update the quantity of the item from the cart component.
+router.put('/:fruitId/:isIncrement', async (req, res, next) => {
+  try {
+    let cart = await getCart(req.user.id)
+
+    if (!cart) {
+      res.status(404).send('Error 404: No Basket')
+      return
```

Comment:
what does return do?

Reply:
return finishes the entire code block and does not continue on. It's a way to prevent too many nested if/else statements.